test_eval_move.py

```diff
@@ -0,0 +1,14 @@
+import state
+import consts
+
+state = state.State()
+state.start_game()
+print("Old position: ", state.active)
+state.move(consts.RIGHT)
+print("New position: ", state.active)
+
+print("Old position: ", state.active)
+state.move(consts.DOWN)
+print("New position: ", state.active)
+
+print("PASS")
```

Comment:
no reason to print "PASS" here since you are manually looking at at new position to see if it was correct. Recommend deleting that print or using an assert before it to automatically check if it was correct.